README.md

```diff
@@ -18,6 +18,25 @@
     - Custom API: `localhost:5000`
     - Simulator API: `localhost:5001`
 
+## Deploy and run with Vagrant on Digital Ocean
```

Comment:
I had to install the DigitalOcean plugin for Vagrant. Maybe we could add the following:
```bash
vagrant plugin install vagrant-digitalocean
```